### PR TITLE
[FIX] #Fixed Typos by codespell

### LINKED ISSUE #347: Typos found by codespell
./tools/xss_attack.py:107: vulnerabilites ==> vulnerabilities
./tools/information_gathering_tools.py:87: Scaning ==> Scanning
./tools/information_gathering_tools.py:117: informations ==> information
./tools/information_gathering_tools.py:168: informations ==> information
./tools/forensic_tools.py:60: Aquire ==> Acquire
./tools/wireless_attack_tools.py:51: bluetooh ==> bluetooth
./tools/webattack.py:89: analizing ==> analyzing
./tools/phising_attack.py:40: enginee ==> engine
./tools/phising_attack.py:95: Engagment ==> Engagement
./tools/payload_creator.py:89: writen ==> written
./tools/others/socialmedia_finder.py:53: Usege ==> Usage

### PR DESCRIPTION
**Pull request by Z4nzu**
Close #347

### CODE REVIEW

**Comment by Z4nzu** [comment]
Hello @cclauss, Please review this branch and approve

**Comment by cclauss** [comment]
Just add this change https://github.com/Z4nzu/hackingtool/issues/347#issuecomment-1454750557) and then if the tests are green, you are good to go.

**Review by cclauss — CHANGES_REQUESTED** [review]
Make codespell a mandatory test https://github.com/Z4nzu/hackingtool/issues/347#issuecomment-1454750557

Don't worry about the failing `push` test.  That is fixed on `master`.

**Comment by Z4nzu** [comment]
> Make codespell a mandatory test [#347 (comment)](https://github.com/Z4nzu/hackingtool/issues/347#issuecomment-1454750557)
> 
> Don't worry about the failing `push` test. That is fixed on `master`.

I did. Please check

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
9 open issues and 7 open pull requests -- Today was spring cleaning!!

File: tools/information_gathering_tools.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ class RedHawk(HackingTool):
 
 
 class ReconSpider(HackingTool):
-    TITLE = "ReconSpider(For All Scaning)"
+    TITLE = "ReconSpider(For All Scanning)"
     DESCRIPTION = "ReconSpider is most Advanced Open Source Intelligence (OSINT)" \
```
Comment (couragexchange) [inline tools/information_gathering_tools.py:88]: ]】

**Review by couragexchange — APPROVED** [review]

**Pull request merged**